lib/services/assets-generation/assets-generation-service.ts

```diff
@@ -55,7 +56,14 @@ export class AssetsGenerationService implements IAssetsGenerationService {
 
 		for (const assetItem of assetItems) {
 			const operation = assetItem.resizeOperation || Operations.Resize;
-			const scale = assetItem.scale || 0.8;
+			let tempScale: number = null;
```

Comment:
In case when `assetItem.scale` is a number, `scale` will always be `0.8` and `assetItem.scale` will be disregarded. Is it the desired behaviour?

If not it should be `let tempScale = assetItem.scale`

Reply:
Thanks , I've fixed it in the code below